### DISCUSSION TITLE
Flash serve on Object Detection task

### DISCUSSION THREAD

**Discussion by IncubatorShokuhou** [PyTorchLightning/lightning-flash#1277] [Q&A]
Is there an exampe on serving an object detection model in lightning-flash?

**Comment by ethanwharris** [comment]
Hi @IncubatorShokuhou thanks for the question! We don't currently support serving for object detection but I've opened #1279 to add it :smiley:

**Reply by IncubatorShokuhou** [reply]
Thank you @ethanwharris 😃
